dbt-cta/twilio/models/0_ctes/calls_ab4.sql

```diff
@@ -0,0 +1,16 @@
+{{ config(
+    cluster_by = "_airbyte_emitted_at",
+    partition_by = {"field": "_airbyte_emitted_at", "data_type": "timestamp", "granularity": "day"},
+    unique_key = '_airbyte_ab_id',
+    tags = [ "top-level-intermediate" ]
+) }}
+-- SQL model to get latest updated dates for each hashid
+-- depends_on: {{ ref('calls_ab3') }}
+select
+    _airbyte_calls_hashid,
+    min(_airbyte_emitted_at) as _airbyte_emitted_at
```

Comment:
should we do max instead of min? wondering if its ever the case where the latest matching hash_id has any new data?

Reply:
Oh yeah, that's definitely something to consider. I'm not really sure the best way to approach this - the logic in the `1_cta_incremental` models seems to be: "add any new rows with more-recent `_airbyte_emitted_at` values". (That's what happens when we partition using _airbyte_emitted_at and set `{{ incremental_clause('_airbyte_emitted_at') }}`, right? 

But probably it makes more sense to have those use `date_updated` (or whatever it's called), huh? And all incremental streams should have some field like that, which we should probably use instead of `_airbyte_emitted_at`.

cc @jackwelty since I know your brain is also in this castle these days